### ISSUE TITLE
words and numbers in thunderbird's notifications

### ISSUE THREAD

**Issue by apan65** [mkiol/GNotifier#146]
Hi, what is this and how to fix it? http://clip2net.com/clip/m11339/0f324-clip-16kb.jpg

**Comment by apan65** [comment]
maybe it because of i'm using beta thunderbird's beta branch?

**Comment by mkiol** [comment]
That is strange.. What is your configuration in "Title for E-mail" and "Text for E-mail"?

![przechwytywanie](https://cloud.githubusercontent.com/assets/1420902/19031563/4ec42a60-8955-11e6-8ae6-9fd052e6c011.PNG)

**Comment by apan65** [comment]
http://clip2net.com/clip/m11339/49930-clip-334kb.jpg

**Comment by mkiol** [comment]
Ok, so I have no idea why there is AC9xxx thing. Let me test it on thunderbird beta and come back to this later.

**Comment by tooomm** [comment]
@apan65 Do you use portable Thunderbird? I realized the same already with portable Installations... (with a normal installation it should say "Mozilla Firefox" or something. Can somebody confirm that?)

Here are my observations:
with 1.9.7
![pic-dl-197](https://cloud.githubusercontent.com/assets/9874850/19044469/baa68ce8-8994-11e6-83e7-71a7adce411b.png)

2min later with version 1.9.8, same Windows, same Firefox
![pic-dl-198](https://cloud.githubusercontent.com/assets/9874850/19044299/fb2938ac-8993-11e6-9c40-fd3674293e23.png)

In the action center you can see that the notification is grouped with that same name/number as well:
![pic-dl-198-2](https://cloud.githubusercontent.com/assets/9874850/19044349/2b205892-8994-11e6-8009-357034b9ffe3.png)

I can find exactly these "id" in the registry as well, so I guess this is a unique name that portable ff/tb and/or your addon uses to register to the windows notification system.

Any way around this? Or how to change the displayed name?
What changed in your addon regarding this? :)

**Comment by apan65** [comment]
Yeah, i use portable thunderbird as well as firefox where everything is fine. 
But you right, i had found these id in registry and change their to thunderbird. Its works, thanks for the help.

**Comment by tooomm** [comment]
I have issues with both portable versions, Firefox and Thunderbird...

What did you exactly change in the registry? Which keys did you change?
Can you give a bit more details? I would happily test it as well!

**Comment by apan65** [comment]
tooomm, regedit - ctrl+f, then you find these words and numbers and change their to "thunderbird" for instance.

**Comment by tooomm** [comment]
@apan65 Did you change all appearances in the registry?

@mkiol Any idea which changes in 1.9.8 make this behavior show up? :)

**Comment by apan65** [comment]
@tooomm yeah

**Comment by mkiol** [comment]
@apan65 @tooomm Honestly, I feel lost right now. So, the "AC9xxx" issue is only on portable versions?

BTW,  I just installed portable FF and GNotifier 1.9.8 doesn't work at all.

**Comment by apan65** [comment]
Yes this only on portable versions

**Comment by tooomm** [comment]
1.9.8 looks like that on my portable TB 45.3.0 as well:
![portable_id](https://cloud.githubusercontent.com/assets/9874850/19093713/7c08028e-8a8c-11e6-88f9-76aaf9c21dad.png)

Where did you download your portable files?

**Comment by mkiol** [comment]
> Where did you download your portable files?

http://portableapps.com/apps/internet/firefox_portable

**Comment by tooomm** [comment]
I use the potable ff version provided by the biggest german tech blog:

http://stadt-bremerhaven.de/portable-firefox-49-fuer-windows/
(the download link at the end of the article is tagged as **"Download Portable Firefox 49 für Windows"**)

**Comment by miLORD1337** [comment]
I use the regular Thunderbird 45.4.0 and the problem is the same. I assume it's related to the way thunderbird installs and uses the registry, as the same number is used to relate the pinned task bar icon to the application itself...
